data/data/aws/vpc/outputs.tf

```diff
@@ -6,6 +6,10 @@ output "vpc_cidrs" {
   value = [data.aws_vpc.cluster_vpc.cidr_block]
 }
 
+output "vpc_ipv6_cidrs" {
```

Comment:
why not use `vpc_cidrs` ?

Reply:
Because the VPC has both IPv4 and IPv6 CIDRs

Reply:
my question was does the output need to make the distinct? can they be a single list?

Reply:
It wouldn't be easy to just use a single list.  They are used as input to different options on a resource: `cidr_blocks` vs `ipv6_cidr_blocks`, so we have to explicitly be using only the IPv4 addresses or only the IPv6 addresses.